framework/core/components/backend.php

```diff
@@ -1018,6 +1055,19 @@ public function _sync_post_separate_meta( $post_id ) {
 
 		return true;
 	}
+	/**
+	 * @param int $term_id
+	 */
+	public function _action_save_taxonomy_fields( $term_id ) {
+		if ( ! isset( $_POST['action'] ) || ! isset( $_POST['taxonomy'] ) ) {
+			return; // this is not real term form submit, abort save
+		}
+		$taxonomy = $_POST['taxonomy'];
+		$custom   = $_POST['fw_options'];
+		foreach ( $custom as $id => $value ) {
+			fw_set_db_term_option( $term_id, $taxonomy, $id, $value );
```

Comment:
Very bad/insecure way to save valued in db, you must use `fw_get_options_values_from_input()`